src/components/_common/office-details/OfficeDetails.module.scss

```diff
@@ -0,0 +1,58 @@
+@use 'src/common' as common;
+
+$margin: 2rem;
+
+.officeDetails {
+    align-self: center;
+    border: none;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    margin-top: 2rem;
+    max-width: common.$contentMaxWidth;
+    width: 100%;
+
+    & > * {
+        width: 100%;
+    }
+}
+
+.pageContent {
+    display: flex;
+    flex-direction: column;
+
+    & > :not(.layout__situation-flex-cols) {
+        margin-bottom: $margin;
+        max-width: common.$sectionMaxWidth;
+    }
+
+    // Prevent margins between adjacent situation-flex-cols layouts
+    .layout__situation-flex-cols + .layout__situation-flex-cols {
+        margin-top: -($margin * 2);
+    }
```

Comment:
`.layout__situation-flex-cols` må ha :global wrapper. Evt kan disse reglene kanskje settes globalt for layout'et, vi har disse repetert i CSS'en for alle sidetyper som bruker dette layout'et såvidt jeg kan se.

Reply:
Etter nærmere ettertanke bør denne bare fjernes. Kontorsidene skal ikke ha noen hosliggende flex-cols uansett.